code/modules/mob/living/carbon/human/species.dm

```diff
@@ -994,7 +994,14 @@
 
 
 /datum/species/proc/disarm(mob/living/carbon/human/user, mob/living/carbon/human/target, datum/martial_art/attacker_style)
-	if(target.check_block())
+	if((user.zone_selected) == "mouth" && (((target.a_intent == INTENT_HELP) && (!target.get_active_held_item())) || (target.restrained()) || (target.zone_selected) == "mouth"))
```

Comment:
Please break this into either one or two if checks or refactor it to be easier to read.

I'm not even sure you need the number of brackets you have